flixel/system/macros/FlxDefines.hx

```diff
@@ -129,6 +131,9 @@ class FlxDefines
 		defineInversion(FLX_NO_FOCUS_LOST_SCREEN, FLX_FOCUS_LOST_SCREEN);
 		defineInversion(FLX_NO_DEBUG, FLX_DEBUG);
 		defineInversion(FLX_NO_POINT_POOL, FLX_POINT_POOL);
+		#if (openfl_legacy && sys || lime >= "8.0.0" && !flash)
+		defineInversion(FLX_NO_PITCH, FLX_PITCH);
+		#end
```

Comment:
the #if shouldn't be here, `FLX_PITCH` and `FLX_NO_PITCH` will always be inversions, you need to define FLX_PITCH [here](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/macros/FlxDefines.hx#L134):

Reply:
why it shouldn't be there?
afaik, defineInversion is for stuff that has 2 sides, the yes and the no version, which, what [`defineInversions()`](https://github.com/HaxeFlixel/flixel/blob/0f3ae2daa75b962956db1ce3b3bac6a299a5755a/flixel/system/macros/FlxDefines.hx#L122) does afaik.
[defineHelperDefines()](https://github.com/HaxeFlixel/flixel/blob/0f3ae2daa75b962956db1ce3b3bac6a299a5755a/flixel/system/macros/FlxDefines.hx#L134) only adds defines depending if it can help the first thing
unless I'm overlooking things

Reply:
None of the other inversion are wrapped in #ifs, take a look at [what 
defineInversion() actually does](https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/macros/FlxDefines.hx#L173). I believe this is the reason flash is failing